frontend/src/lib/doc-util.tsx

```diff
@@ -19,14 +19,12 @@ export const DOC_LINKS = {
     CREATE_CONNECTION_VMWARE: `${DOC_BASE_PATH}/multicluster_engine/index#creating-a-credential-for-vmware-vsphere`,
     CREATE_CONNECTION_OPENSTACK: `${DOC_BASE_PATH}/multicluster_engine/index#creating-a-credential-for-openstack`,
     CREATE_CONNECTION_VIRTUALIZATION: `${DOC_BASE_PATH}/multicluster_engine/index#creating-a-credential-for-virtualization`,
-    CREATE_CONNECTION_BAREMETAL: `${DOC_BASE_PATH}/multicluster_engine/index#credentials`,
     CREATE_CONNECTION_ANSIBLE: `${DOC_BASE_PATH}/multicluster_engine/index#creating-a-credential-for-ansible`,
     CREATE_CONNECTION_REDHATCLOUD: `${DOC_BASE_PATH}/multicluster_engine/index#creating-a-credential-for-openshift-cluster-manager`,
     CREATE_CONNECTION_PROXY: `${DOC_BASE_PATH}/multicluster_engine/index#cluster-overview`,
     CREATE_CONNECTION_PROXY_VMWARE: `${DOC_BASE_PATH}/multicluster_engine/index#proxy-vm`,
     CREATE_CONNECTION_PROXY_AWS: `${DOC_BASE_PATH}/multicluster_engine/index#proxy-aws`,
     CREATE_CONNECTION_PROXY_AZURE: `${DOC_BASE_PATH}/multicluster_engine/index#proxy-azure`,
-    CREATE_CONNECTION_PROXY_BAREMETAL: `${DOC_BASE_PATH}/multicluster_engine/index#proxy-bare`,
```

Comment:
FYI @cadawson 
BMA link removal.

Reply:
https://github.com/stolostron/console/pull/2063/files#diff-235b60184cc6834bb0c45e7a9e3a7e0731dd01d475e87028e43f06547f243808L22

This line as well.